code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -102,13 +100,12 @@ tNT = uvc "tNT" (nounPhraseSP "TNT equivalent factor")
 
 standOffDist = uq (constrained' (uc sD (variable "SD") Real metre)
   [ gtZeroConstr,
-    sfwrc $ Bounded (Inc, sy sdMin) (Inc, sy sdMax)] (exactDbl 45)) defaultUncrt
+    sfwrRange $ Bounded (Inc, sy sdMin) (Inc, sy sdMax)] (exactDbl 45)) defaultUncrt
 
 nomThick = cuc "nomThick" 
-  (nounPhraseSent $ S "nominal thickness" +:+ displayDblConstrntsAsSet 
-    nomThick nominalThicknesses)
+  (nounPhraseSent $ S "nominal thickness t is in" +:+ eS (Spc $ DiscreteD nominalThicknesses))
   lT millimetre {-Discrete nominalThicknesses, but not implemented-} Rational 
-  [{- TODO: add back constraint: enumc nominalThicknesses -}] $ exactDbl 8
+  [sfwrElem $ mkSet (map dbl nominalThicknesses)] $ exactDbl 8 -- for testing
```

Comment:
Right, that's good -- but I don't see, other than in the code, where this information re-appears in the SRS itself. I only see it disappear!

Reply:
It probably would be good to show it somewhere in the SRS too. I think we have a constraints table that it is supposed to be shown in. @NoahCardoso can you confirm please? CCing @smiths because he probably has suggestions too.

Reply:
I added back the constraint to the definition of nomThick

Reply:
I've looked at the current location of the `nomThick` information and it looks fine to me.  It would be nice in the future if our SRS let us define types and then nominal thickness could be a type.